README.md

```diff
@@ -22,7 +21,7 @@ console.log(results);
 ```
 
 ```js
-// Your answer...
+var results = mongoose.model('Bob', AuthorModel);
```

Comment:
You need to wrap the `console.log` inside of a callback or promise since Mongoose queries are asynchronous.